### ISSUE TITLE
KSP 1.3.0 - Current dev version of GPP not loading properly

### ISSUE THREAD

**Issue by mas1701** [Galileo88/Galileos-Planet-Pack#15]
Hi...

I'm  testing the currently development version of GPP on KSP 1.3.0 (x64, Windows), but it won't load properly. While i can get to the KSP main screen, I cannot open settings or start / resume a game, it always ends in an endless loading loop.

Moving GPP out of GameData is the only way to solve the problem, even when using only the minimum of required mods.

I'm adding my log file.

I hope this can be fixed and that my reports helps fixing it.

[LOG FILE](http://adblock.mscs.net/ksp-20170808.txt)

**Comment by theRagingIrishman** [comment]
Seriously bro?

**Comment by theRagingIrishman** [comment]
"CASUAL PLAYERS: DO NOT DOWNLOAD FROM HERE. ONLY DOWNLOAD FROM THE RELEASES PAGE. THE DEV VERSION IS NOT RELEASE-READY AND WILL DO MORE HARM THAN GOOD TO YOUR PLAYTHROUGH."

**Comment by theRagingIrishman** [comment]
^ from the readme

**Comment by theRagingIrishman** [comment]
also, it works 100% fine on my end so the error is definitely between the chair and the computer

**Comment by mas1701** [comment]
Good morning Sir theRagingIrishman,

I'm sorry, but I was unaware that a bug report about an openly availible dev version or beta product justifies extreme rudeness and offensive behavior.

As an IT systems and network administrator I'm used to developers being quite happy when they get feedback from early adopters, so the product is well-tested on final release. I'm not used to be lectured about what ordinary players should do, neither am I used to being accused to have done something wrong without even looking at the log information provided. I'm in no way implying that I don't make mistakes myself, but this answer suggests that it must have been the case here, just because the problems don't exist on ONE other system you're talking about.

I don't dare to judge about your development skills here, but your social skills seem to have fallen short.

Best regards

Mark

**Comment by theRagingIrishman** [comment]
@mas1701 I'm very sorry about the way I responded. I was tired and in a bad mood and it was way out of line.

**Comment by eiktyrner** [comment]
For what it's worth, I get the same issue with the current master branch. Just wanted to contribute with more data if this is an issue that is only visible for some people. 

Installed fresh from Steam and followed the steps outlines above. 

**KSP.log** https://gist.github.com/eiktyrner/913b5680016cc6698be427ea251a62de
![ksp_gpp](https://user-images.githubusercontent.com/4710252/29088159-9ad6d7c4-7c78-11e7-83bd-bdb8f33f7c24.PNG)
![ksp_gpp2](https://user-images.githubusercontent.com/4710252/29088160-9af80458-7c78-11e7-94d9-7c6549eff88f.PNG)

**Comment by theRagingIrishman** [comment]
Hmm @mas1701 @eiktyrner you guys are having the same error but I don't know what's causing it...
@Galileo88 any thoughts?

**Comment by Galileo88** [comment]
Ugh. Ok, the issue most seem to be having is they are not downloading the correct textures or don't know where to find them. The textures available for 1.2.2 do not work with the 1.3. You will have to use the dev branch for the textures. If you don't know where to find that, I'm sorry, keep using 1.2.2. If you are downloading the textures from the "releases" you are downloading the wrong textures.

Also, if you are running a 100% fresh version of ksp, you must run it once without any mods. Then you can add GPP.

The dev version is not for public use and I won't supply any more support beyond this. 

This is what I was afraid of letting a people use the dev version for youtube.

**Comment by mas1701** [comment]
Hi folks ...

solution found! Also for @eiktyrner !

tRI> @mas1701 I'm very sorry about the way I responded.
tRI> I was tired and in a bad mood and it was way out of line. Now for a proper response:

Apology accepted. I guess this something that can happen, although it shouldn't. ;-)

tRI> Just so you know, [...] . The reason I strongly suspect
tRI> an installation error is that all of us devs load it up
tRI> fairly often and we all have very different devices.
tRI> Is there anything special about your computer?

Actually, no. It is not an installation error, not PEBKAC (Problem Exists Between Keyboard And Chair) and not anything special about the computer. It's the installation instructions I followed before. Instructions by the GPP team here on GitHub. And it's the exact same thing that happened to @eiktyrner . I'll explain below.

tRI> Were you forcing anything when opening KSP?

Nope. Unless you count the warning, that GPP is for KSP v1.2.2. It an incorrect warning, obviously.

tRI> I'd suggest starting over on a clean install of KSP 1.3.

Of course!

_For every new modded install, I fork a new KSP folder structure, symlinking shared folders to the master (stock) install and keeping only "CKAN" Information, "GameData", "Logs", "saves", "Screenshots" and "ships" folders separately. This is a proven method of saving a lot of (SSD) disk space, when using multiple configurations. And yes, that's using MS Windows (Windows 10 Pro), Windows can handle symlinks for files and folders quite well since Windows 7 (they "stole" that idea from Unix/Linux ;-) ). If you are curious, take a look at the command line tool "mklink". It has nothing to do with explorer shortcuts, btw.!_

tRI>     Install a clean copy of KSP 1.3

Check!

tRI>     Run it once before installing any mods

Check!

tRI>     Install Kopernicus 1.3.0-4 [..]

Check!

tRI>     Install ModuleManager 1.8.1 [...]

Check! (Previously, I also tried 1.8.0 as part of my debugging attempts)

Btw., you forgot to mention to replace the old DLL with the new one, as Kopernicus already includes an older ModuleManager. ;-)

tRI>     Install the GPP folder from the master branch

Check!

tRI>     Install GPP_Textures from the textures branch
tRI> (folder path should be /GameData/GPP/GPP_Textures)

Whoops! That instruction differs from the instructions at:
https://github.com/Galileo88/Galileos-Planet-Pack
(automatically displayed from README.md)

README.md says:

> 6. Download GPP_Textures.

It links to GPP_Textures at: https://github.com/Galileo88/Galileos-Planet-Pack/releases/tag/1.0

And that is an old version! I compared it with your instructions to download textures from the textures branch and there is BIG a difference! As you can see on the screenshot posted by @eiktyrner , the old texture pack has no subfolders inside of GPP_Textures. My newly downloaded textures from the texture branch also include a subfolder named PluginData. And it makes a hell of a difference!

So, the correct texture files required are located here:
https://github.com/Galileo88/Galileos-Planet-Pack/archive/GPP_Textures.zip

That will pull the right stuff for the dev version. And that is why all your dev installs work just fine - you know about this, everyone else who's following the README.md doesn't!

tRI>     Run KSP

Check!

Old textures: endless loading loop, lots of errors
New textures: loads just fine! (I already took a glance at the wonderful new planets and moons)

tRI> If that doesn't work, log file again as well as a screenshot
tRI> of the contents of you GameData and GPP folders

No need any more.

@eiktyrner : Delete the old textures, get the new ones, then you should be fine as well.

Yours
Mark

**Comment by mas1701** [comment]
@Galileo88 

> This is what I was afraid of letting a people use the dev version for youtube.

This is what happens when you're using an open online repository that everyone can find and when the instructions you post there yourself (as a dev team) are just wrong. It has NOTHING to do with YouTube.

At least, now you know how this has happened and that the dev version runs quite fine so far if installed correctly. Don't blame others, be thankful instead.

Mark, signing off.

**Comment by Galileo88** [comment]
Ok, Mark. You really think our instructions are going to be for how to install the dev version? NO! Especially when we say NOT TO USE IT! Of course the instructions don't point to the texture branch. I even said that in my one comment on here and told you that you needed the textures from the textures branch. The issue here was clearly PEBKAC. Our instructions are accurate for when GPP releases. The link for the textures will obviously change when there is a release for the new textures. It's called a placeholder. Do not come to my repo, screw up an istall, because you are incompetent in modding ksp, and tell me, I should be thankful. I didn't put up flyers asking for testers. We have been doing this for a while now. You may have worked IT, but you clearly have no experience with project management. Screw off.

**Comment by Galileo88** [comment]
And believe me, if github would allow private repos with public releases, it would have been done a long time ago. That way, I wouldn't have to deal with people applying lipstick to their impatience, and calling it "testing".

**Comment by mas1701** [comment]
@Galileo88 

OK, bad social skills here, too. Otherwise you would refrain from using terms like "fuck off". That alone disqualifies yourself, but never mind. If you think, aggressive behaviour will make anything better, so be it. Maybe you'd qualify as the next US president, it doesn't seem to require much diplomatic skills either.

Have you, at any time, thought about the small amount of time between our two postings? I didn't even read your posting explaining the situation (your posting beginning with "ugh") before sending in my own findings, because when I started typing, your posting didn't even exist.

The instructions I followed are from the master branch:

> CASUAL PLAYERS: DO NOT DOWNLOAD FROM HERE. ONLY DOWNLOAD FROM THE RELEASES PAGE. THE DEV VERSION IS NOT RELEASE-READY AND WILL DO MORE HARM THAN GOOD TO YOUR PLAYTHROUGH.
Galileos-Planet-Pack

This mods is licensed by Creative Commons Attribution-NonCommercial-NoDerivs CC BY-NC-ND

TO INSTALL:

    Create a fresh install of KSP version 1.3.0. Ensure that KSP always launches in 64-bit. Run once with no mods installed.

    Download the third-party mod Kopernicus. The Kopernicus version number must match the KSP version number, i.e. 1.3.0.

    Install by copying from [Kopernicus Download]\GameData\ to [KSP]\GameData\ the following folders and files:
        Kopernicus\
        ModularFlightIntergrator\
        ModuleManager.2.8.0.dll

    Download Galileo’s Planet Pack 1.4.0.

    Copy from [GPP Download]\GameData\ to [KSP]\GameData\ the folder GPP\ and all its contents.

    Download GPP_Textures.

    Copy from [Textures Download]\GameData\GPP\ to [KSP]\GameData\GPP\ the folder GPP_Textures.

    Run once in this configuration before installing additional mods

----

While these instructions are from the master branch, they explicitly state that they are written for KSP 1.3.0. They also refer to a GPP version 1.4.0, that isn't even released yet, so it's a fair assumption, that it refers to the future version now being developed. PEBKAC? I don't think so. Not on my end, that is.

And btw., I AM working as an IT systems admin, it's not "I WAS". Just yesterday, we had an internal meeting in the company (developing web solutions for car manufacturers) and the CEO was very pleased with a new project that is now ready for use. A fine collaboration work by developers and administrators, three people working together. So I can't work together with developers as you hinted? Really?

Maybe you are used to judge everyone reporting bugs here to be "simple, stupid", or as you call it "casual" players, but obviously, not all of them are.

I will no longer participate in this conversation. I've been working with IT systems for 30 years, I don't need this childish stuff. Grow up!
